.travis.yml

```diff
@@ -41,7 +44,14 @@ script:
   - diff -B phpcs.mo4.xml <(xmllint --format phpcs.mo4.xml)
   - diff -B phpcs.xml.dist <(xmllint --format phpcs.xml.dist)
   - vendor/bin/phpcs -s --standard=MO4 integrationtests/testfile.php # Check, if all the sniffs work with the current dependencies
-  - vendor/bin/phpunit # Run tests
+  # Another PHPUnit workaround, this time, because of PHPCS itself
+  - |
+    if [ "${TRAVIS_PHP_VERSION}" != "8.1.0" ]; then
+      export NEXT_PHPUNIT_WORKAROUND=""
```

Comment:
I suggest to simplify the new `if-then-elif ` to a simple `if-then-else`
```
if [ "${TRAVIS_PHP_VERSION}" = "8.1.0" ]; then
    export NEXT_PHPUNIT_WORKAROUND="--no-configuration --bootstrap=tests/bootstrap.php --dont-report-useless-tests vendor/squizlabs/php_codesniffer/tests/Standards/AllSniffs.php"
else
    export NEXT_PHPUNIT_WORKAROUND=""
fi
```

Reply:
Should be addressed now.